ata_api/main.py

```diff
@@ -0,0 +1,47 @@
+from os import getenv
+from random import randint
+
+from ata_db_models.helpers import get_conn_string
+from ata_db_models.models import Prescription
+from fastapi import FastAPI
+from mangum import Mangum
+from sqlmodel import Session, create_engine, select
+
+app = FastAPI()
+engine = create_engine(url=get_conn_string())
+STRATEGY = getenv("STRATEGY", "RCT")
```

Comment:
⛏ Is it possible to change "strategy" to "policy," for the sake of consistency with the experiment doc?